com.github.expresslrs.ExpressLRSConfigurator.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.github.expresslrs.ExpressLRSConfigurator
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: expresslrs-configurator
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --socket=system-bus
+  - --filesystem=/tmp
+  - --filesystem=/run/udev:ro
+  - --filesystem=~/.platformio
+modules:
+  - name: git
+    make-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    make-install-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    post-install:
+      - find . -type f -name perllocal.pod -delete
+    sources:
+      - type: archive
+        sha256: 814641d7f61659cfbc17825d0462499ca1403e39ff53d76a8512050e6483e87a
+        url: https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.3.tar.xz
+        x-checker-data:
+          type: html
+          url: https://mirrors.edge.kernel.org/pub/software/scm/git/
+          version-pattern: git-([\d\.-]+).tar.xz
+          url-template: https://mirrors.edge.kernel.org/pub/software/scm/git/git-$version.tar.xz
+  - name: expresslrs-configurator
+    buildsystem: simple
+    build-commands:
+      - ar x expresslrs-configurator_*.deb
+      - rm expresslrs-configurator_*.deb
+      - tar xf data.tar.xz
+      - cp -r 'opt/ExpressLRS Configurator' '/app/ExpressLRS-Configurator'
+      - mkdir -p /app/share/icons/hicolor
+      - chmod -R a-s,go+rX,go-w "/app/ExpressLRS-Configurator"
+      - install expresslrs-configurator.sh /app/bin/expresslrs-configurator
+      - install -Dm644 expresslrs-configurator.png /app/share/icons/hicolor/scalable/apps/com.github.expresslrs.ExpressLRSConfigurator.png
+      - install -Dm644 com.github.expresslrs.ExpressLRSConfigurator.desktop /app/share/applications/com.github.expresslrs.ExpressLRSConfigurator.desktop
+      - install -Dm644 com.github.expresslrs.ExpressLRSConfigurator.appdata.xml /app/share/metainfo/com.github.expresslrs.ExpressLRSConfigurator.metainfo.xml
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/ExpressLRS/ExpressLRS-Configurator/releases/download/v1.5.3/expresslrs-configurator_1.5.3_amd64.deb
+        x-checker-data:
+          type: anitya,
+          project-id: 301823,
+          url-template: https://github.com/ExpressLRS/ExpressLRS-Configurator/releases/download/$version/expresslrs-configurator_${version}_amd64.deb,
+          is-main-source: true
+        sha256: ecb82c07a620fc21286bf92161722360090974a47b84ea6fe55490ca5456d610
+      - type: file
+        path: expresslrs-configurator.sh
+      - type: file
+        path: com.github.expresslrs.ExpressLRSConfigurator.desktop
```

Comment:
do you really need these file since you are extracting the .deb binary?

Reply:
I at least couldn't find these, but that might be due to me not knowing, what I do.

Reply:
```shell
[hub@galactica expresslrs]$ ar xf expresslrs-configurator_1.5.3_amd64.deb 
[hub@galactica expresslrs]$ tar Jtvf data.tar.xz |grep desktop
-rw-r--r-- 0/0             302 2022-11-20 08:03 ./usr/share/applications/expresslrs-configurator.desktop
```

right there.